tests/src/test/java/org/jboss/windup/tests/application/WindupArchitectureSmallBinaryModeTest.java

```diff
@@ -66,17 +66,17 @@ public void testRunWindupTiny() throws Exception
             Path indexPath = graphDirectory.resolve(Paths.get("index.html"));
 
             Path appReportPath = resolveChildPath(reportsDirectory,
-                        "JEE_Example_App__org_windup_example_jee_example_app_1_0_0_\\.\\d+\\.html");
+                        "JEE_Example_App__org_windup_example_jee_example_app_1_0_0_\\.html");
             Path appNonClassifiedReportPath = resolveChildPath(reportsDirectory,
-                        "nonclassifiedfiles_JEE_Example_App__org_windup_example_jee_example_app_1_0_0_\\.\\d+\\.html");
-            Path productCatalogBeanPath = resolveChildPath(reportsDirectory, "ProductCatalogBean_java\\.\\d+\\.html");
-            Path loginFilterPath = resolveChildPath(reportsDirectory, "LoginFilter_java\\.\\d+\\.html");
-            Path loginEventPublisherPath = resolveChildPath(reportsDirectory, "LogEventPublisher_java\\.\\d+\\.html");
-            Path authenticationFilterPath = resolveChildPath(reportsDirectory, "AuthenticateFilter_java\\.\\d+\\.html");
+                        "nonclassifiedfiles_JEE_Example_App__org_windup_example_jee_example_app_1_0_0_\\.html");
+            Path productCatalogBeanPath = resolveChildPath(reportsDirectory, "ProductCatalogBean_java\\.html");
+            Path loginFilterPath = resolveChildPath(reportsDirectory, "LoginFilter_java\\.html");
```

Comment:
Did you add any tests to ensure that the new hint features do not break? I think I'm OK with merging without tests, because we need to go in and figure out a way to test the rules better anyway, but, just sayin' :)